core/invoker/src/main/resources/application.conf

```diff
@@ -71,6 +71,8 @@ whisk {
     prewarm-promotion: false # if true, action can take prewarm container which has bigger memory
     memory-sync-interval: 1 second # period to sync memory info to etcd
     batch-deletion-size: 10 # batch size for removing containers when disable invoker, too big value may cause docker/k8s overload
+    # optional setting to specify the total allocatable cpus for all action containers, each container will get a fraction of this proportional to its allocated memory to limit the cpu
+    # user-cpus: 1
```

Comment:
BTW, could you add a configuration like this?
https://github.com/apache/openwhisk/blob/master/ansible/roles/invoker/tasks/deploy.yml#L304

It would be easier to configure it with ansible.

Reply:
@style95 I just pushed some changes to `ansible/roles/invoker/tasks/deploy.yml` and `ansible/group_vars/all`. I'm not super familiar with ansible and was trying to keep it optional. Can you let me know how that looks?